test/edu-benefits/pages/schoolSelection.unit.spec.jsx

```diff
@@ -1,141 +1,76 @@
-
-describe('Edu 1990e schoolSelection', () => {
-  const { schema, uiSchema } = formConfig(fullSchema1990e, {
-    fields: [
-      'educationProgram',
-      'educationObjective',
-      'nonVaAssistance',
-      'civilianBenefitsAssistance'
-    ]
-  });
-
-  it('should render', () => {
-    const form = ReactTestUtils.renderIntoDocument(
-      <DefinitionTester
-          schema={schema}
-          data={{}}
-          definitions={formConfig.defaultDefinitions}
-          uiSchema={uiSchema}/>
-    );
-
-    const formDOM = findDOMNode(form);
-
-    const inputs = Array.from(formDOM.querySelectorAll('input, select, textarea'));
-
-    expect(inputs.length).to.equal(7);
-  });
-
-  it('should have no required inputs by default', () => {
-    const onSubmit = sinon.spy();
-    const form = ReactTestUtils.renderIntoDocument(
-      <DefinitionTester
-          schema={schema}
-          onSubmit={onSubmit}
-          data={{}}
-          definitions={formConfig.defaultDefinitions}
-          uiSchema={uiSchema}/>
-    );
-    const formDOM = findDOMNode(form);
-    submitForm(form);
-
-    expect(Array.from(formDOM.querySelectorAll('.usa-input-error'))).to.be.empty;
-    expect(onSubmit.called).to.be.true;
-  });
-
-  it('should require educationType if specified', () => {
-    const { schema: schemaIfEducationType, uiSchema: uiSchemaIfEducationType } = formConfig(fullSchema1990e, {
-      fields: [
-        'educationProgram',
-        'educationObjective',
-        'nonVaAssistance',
-        'civilianBenefitsAssistance'
-      ],
-      required: ['educationType']
+import { schoolSelectionOptionsFor as optionsFor } from '../../../src/js/edu-benefits/pages/schoolSelection.js';
+
+import formConfig1990e from '../../../src/js/edu-benefits/1990e/config/form';
+import formConfig1990n from '../../../src/js/edu-benefits/1990n/config/form';
+import formConfig5490 from '../../../src/js/edu-benefits/5490/config/form';
+
+// Actually use the configs from the forms
+const configs = {
+  '1990n': formConfig1990n,
+  '1990e': formConfig1990e,
+  '5490': formConfig5490 // eslint-disable-line
+};
+
+// Describe the tests for each form
+Object.keys(optionsFor).forEach((formName) => {
+  describe(`Edu ${formName} schoolSelection`, () => {
+    // This is a bit fragile; requires the chapter and page name to be schoolSelection
+    const { schema, uiSchema } = configs[formName].chapters.schoolSelection.pages.schoolSelection;
+
+    // They should all render
+    it('should render', () => {
+      const form = ReactTestUtils.renderIntoDocument(
+        <DefinitionTester
+            schema={schema}
+            data={{}}
+            definitions={configs[formName].defaultDefinitions}
+            uiSchema={uiSchema}/>
+      );
+
+      const formDOM = findDOMNode(form);
+
+      const inputs = Array.from(formDOM.querySelectorAll('input, select, textarea'));
+
+      // Should have a better test than this...
```

Comment:
You could define an object above and hard-code the expected number of inputs for each form, and use that here.